InomialClient.js

```diff
@@ -33,13 +40,31 @@ class InomialClient {
     // Origin is nullable is the WSS Origin header; should be null for web clients, or
     // you can set it to let the server know what your application name is.
     //
+    // websocketClientConfig is an optional JS object passed-on verbatim to the WebSocketClient
+    // constructor; this can allow the caller to fine-tune socket/TLS parameters as needed.
+    // See <https://github.com/theturtle32/WebSocket-Node/blob/master/docs/WebSocketClient.md#client-config-options>
+    // for more details on what properties this object can accept.
     //
-    constructor(hostname, stage, origin, apikey)
+    // logLevel (if specified) should be one of the following constants:
```

Comment:
at most we should have a "verbose" flag here, which should also be available from the environment.